### PR TITLE
Feat examples

### PR DESCRIPTION
**Pull request by rumpl**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# syntax = docker/dockerfile:experimental
+FROM node:12-alpine as base
+ARG DOCKER_GITHUB_TOKEN
+ENV DOCKER_GITHUB_TOKEN=${DOCKER_GITHUB_TOKEN}
+COPY package.json .
+COPY yarn.lock .
+
+RUN --mount=type=cache,target=cache \
```
Comment (chris-crone) [inline Dockerfile]: Is this the right cache directory for `yarn`? I'm not sure where it mounts if you don't specify the absolute path

**Review by gtardif — APPROVED** [review]
and build is quite quick !

File: src/index.ts [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
 import { credentials } from '@grpc/grpc-js';
 import { ContainersClient } from './protos/containers/v1/containers_grpc_pb';
 import { ContextsClient } from './protos/contexts/v1/contexts_grpc_pb';
 import { ComposeClient } from './protos/compose/v1/compose_grpc_pb';
 import { StreamingClient } from './protos/streams/v1/streams_grpc_pb';
 
-// ~/Library/Containers/com.docker.docker/Data/cli-api.sock
-const address = 'unix:///tmp/backend.sock';
-
+let addr = 'unix:///./pipe/dockerCliApi';
```
Comment (gtardif) [inline src/index.ts]: Ah, it's 4 slashes in this windows url... `unix:////./pipe/dockerCliApi`

**Review by gtardif — CHANGES_REQUESTED** [review]
Just a missing slash on the npipe url

File: src/index.ts (hunk 2 above)
Comment (rumpl) [reply src/index.ts]: Fixed, thanks, I removed it thinking it was a typo

File: Dockerfile (hunk 1 above)
Comment (rumpl) [reply Dockerfile]: Fixed

**Review by chris-crone — APPROVED** [review]
LGTM

**Review by gtardif — APPROVED** [review]

**Pull request merged**